zookeeper/source/zk_client_common.cc

```diff
@@ -25,6 +25,7 @@ namespace zkclient {
     const std::string ZkClientCommon::CLASS_NAME = ": **ZkNnCommon** : ";
 	const std::string ZkClientCommon::BLOCK_LOCATIONS = "/block_locations/";
     const std::string ZkClientCommon::BLOCKS = "/blocks";
+    const std::string ZkClientCommon::DELETE_BACKSLASH = "/delete/";
```

Comment:
although now that you mention it i think it's right to delete it since it's never used